### DISCUSSION TITLE
LHR 3070 ,DaggerAutokylos Terminated when trying to run the benchmark

### DISCUSSION THREAD

**Discussion by Luckydepressedguy** [angelbbs/NiceHashMinerLegacy#124] [General]
+0 core +1000 mem 82% power limit and  70% fan speed. 
Is there any way I can dual mine with nicehash Legacy Fork Fix 42 ?

**Comment by isoneguy** [comment]
Have you tried running at +0 core with 83% power limit and 70% fan speed?

all claymore needs is to be updated to the current epoch to support dual mining.
